provisioning/provisioning-samples/service-bulkoperation-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/ServiceBulkOperationSample.java

```diff
@@ -28,63 +45,91 @@
         {
-            String registrationId = individualEnrollment.getRegistrationId();
+            String registrationId = individualEnrollment.registrationId();
             System.out.println("\nGet the individualEnrollment information for " + registrationId + ":");
             IndividualEnrollment getResult = provisioningServiceClient.getIndividualEnrollment(registrationId);
-            System.out.println(getResult);
+            System.out.println(ow.writeValueAsString(getResult));
+            getResult.withDeviceId("java_device_id");
+            updateEnrollments.add(getResult);
         }
+        bulkEnrollmentOperation = new BulkEnrollmentOperation()
+        		.withEnrollments(updateEnrollments)
+        		.withMode(BULK_OPERATION_UPDATE);
+        System.out.println("\nRun the bulk operation to update the individualEnrollments...");
+        bulkOperationResult =  provisioningServiceClient.runBulkEnrollmentOperation(bulkEnrollmentOperation);
+        System.out.println("Result of the Update bulk enrollment...");
+        System.out.println(bulkOperationResult.isSuccessful());
 
         // ************************************ Query info of individualEnrollments ***********************************
-        System.out.println("\nCreate a query for individualEnrollments...");
+        System.out.println("\nCreate a query for enrollments...");
         QuerySpecification querySpecification =
-                new QuerySpecificationBuilder("*", QuerySpecificationBuilder.FromType.ENROLLMENTS)
-                        .createSqlQuery();
-        Query query = provisioningServiceClient.createIndividualEnrollmentQuery(querySpecification, QUERY_PAGE_SIZE);
-
-        while(query.hasNext())
+                new QuerySpecification().withQuery("SELECT * FROM ENROLLMENTS");
+        List<IndividualEnrollment> queryResult = provisioningServiceClient.queryIndividualEnrollments(querySpecification);
+        
+        for (IndividualEnrollment eachEnrollment : queryResult)
         {
-            System.out.println("\nQuery the next individualEnrollments...");
-            QueryResult queryResult = query.next();
-            System.out.println(queryResult);
+        	System.out.println(ow.writeValueAsString(eachEnrollment));
         }
 
         // ********************************** Delete bulk of individualEnrollments ************************************
         System.out.println("\nDelete the set of individualEnrollments...");
-        bulkOperationResult =  provisioningServiceClient.runBulkEnrollmentOperation(BulkOperationMode.DELETE, individualEnrollments);
-        System.out.println(bulkOperationResult);
+        bulkEnrollmentOperation = new BulkEnrollmentOperation()
+        		.withEnrollments(individualEnrollments)
+        		.withMode(BULK_OPERATION_DELETE);
+        bulkOperationResult =  provisioningServiceClient.runBulkEnrollmentOperation(bulkEnrollmentOperation);
+        System.out.println(bulkOperationResult.isSuccessful());
```

Comment:
System.out.println(bulkOperationResult.isSuccessful() ? "Bulk enrollment deletion was successful!" : "Bulk enrollment deletion was not successful"); to be a bit more descriptive

Reply:
same in the above Println statements where you are just printing a boolean with no other context

Reply:
yeah, I agree; I seem to have missed the description for the delete operation